tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/builder/BallerinaASTModelBuilder.java

```diff
@@ -282,4 +283,28 @@ public void addComment(String comment) {
     public void addFunctionInvocationStatement(boolean argsAvailable) {
         modelBuilder.createFunctionInvocationStmt(null, null, nameReferenceStack.pop(), argsAvailable);
     }
+
+    public void createWorkerInvocationStmt(String workerName) {
+        modelBuilder.createWorkerInvocationStmt(workerName, null, null);
+    }
+
+    public void enterWorkerDeclaration() {
+        //  isWorkerStarted = true;
```

Comment:
let's remove commented code

Reply:
This was fixed in this commit >> https://github.com/wso2/product-ei/pull/979/files#diff-e926d4a6b8bdff790cea82e4ff1ce334L329